mw-webapp/src/component/dropdown/dropdownMenuItem/DropdownMenuItem.module.scss

```diff
@@ -1,14 +1,24 @@
 @import "src/styles/variables";
 
 .dropdownMenuItem {
-  padding: $paddingSmall;
+  padding: $paddingSmall $paddingMedium;
+  font-size: $fontSizeMedium;
+  cursor: pointer;
 
-  &:hover {
-    background-color: var(--secondaryBgColor);
-    cursor: pointer;
+  &:not(:last-child) {
+    border-bottom: $primaryBorder solid var(--tertiaryBgColor);
+  }
+  &:first-child {
+    border-top-left-radius: $primaryBorderRadius;
+    border-top-right-radius: $primaryBorderRadius;
+  }
+  &:last-child {
+    border-bottom-left-radius: $primaryBorderRadius;
+    border-bottom-right-radius: $primaryBorderRadius;
   }
 
+  &:hover,
   &:active {
-    background-color: var(--generalPrimaryColor);
+    color: var(--selectHoverColor);
```

Comment:
dropdown is not select

Reply:
![image](https://github.com/tritonJS826/masters-way/assets/112805668/b6b1d201-5714-4438-8d1f-47c0941dc448)

Reply:
fixed